### PR TITLE
Fix Jenkins builds

### PR DESCRIPTION
**Pull request by J-N-K**
Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

**Comment by J-N-K** [comment]
I noticed that my Jenkins downloads `org.minidns/minidns.core/0.3.4` instead of `org.minidns/minidns.core/0.3.3`. My local build does that too, but the verification succeeds. With this change locally 0.3.3 is downloaded. Let's see if this makes any difference.

The same is true for the mqttembeddedbroker, it downloads 4.1.22.Final were 4.1.34.Final is requested in the feature.xml. There is no dependency defined in the pom, so maybe that is the culprit.

**Comment by J-N-K** [comment]
I still don't understand why this is necessary (as Travis and the local build do not suffer from this problem), but it fixed it and is not wrong.

File: bundles/org.openhab.io.mqttembeddedbroker/pom.xml [hunk 1]
```diff
@@ -41,5 +41,48 @@
...
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-mqtt</artifactId>
+      <version>4.1.34.Final</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-resolver</artifactId>
+      <version>4.1.34.Final</version>
```
Comment (martinvw) [inline bundles/org.openhab.io.mqttembeddedbroker/pom.xml]: Would it make sense to extract the repeated version number? Is it expected that we need to add all of them here?

Comment (J-N-K) [reply bundles/org.openhab.io.mqttembeddedbroker/pom.xml]: Done. As I said: I don't fully understand why it is necessary to add them here as the build didn't require it before splitting the features into seperate files. Maybe it depends on the build order which version is downloaded.

All of these dependencies were declared in the feature before, I only added them to the POM to pin the downloaded version.

**Pull request merged**